docs/_static/renderjson.js

```diff
@@ -0,0 +1,194 @@
+// Copyright © 2013-2014 David Caldwell <david@porkrind.org>
```

Comment:
This code is also in the `_static` directory in the main standard (https://github.com/open-contracting/standard/blob/26075ede08503e3e10e097af20b40cc85107d4cc/standard/docs/en/_static/renderjson.js)

The paragraph after the copyright notice states:

> // Permission to use, copy, modify, and/or distribute this software for any
> // purpose with or without fee is hereby granted, provided that the above
> // copyright notice and this permission notice appear in all copies.

Does that address your concerns?

Reply:
Since it's only one file (we authored the CSS file), we can leave it as-is.

Please add a comment to the CSS file:

`# See https://github.com/open-contracting/standard/blob/1.1-dev/standard/docs/en/_static/renderjson.css

Reply:
Added in https://github.com/open-contracting/infrastructure/pull/72/commits/c93a5b9bcfc04da9a405f4b2569f66d5ed3b9752